tests/grammar/service-shapes.smithy

```diff
@@ -1,4 +1,4 @@
-// SYNTAX TEST "source.smithy" "This tests resource shapes"
+// SYNTAX TEST "source.smithy" "This tests service shapes"
```

Comment:
Do we need a grammar test for resource properties?

Reply:
Seems like a separate PR before the next release as well.